src/main/java/org/mule/tools/apikit/model/MuleDomain.java

```diff
@@ -44,6 +53,38 @@ public static MuleDomain fromInputStream(InputStream content) throws Exception {
     return new MuleDomain(content, httpListenerConfigs);
   }
 
+  public static MuleDomain fromConfigFile(File configFile) throws Exception {
+    try (InputStream configFileIS = new FileInputStream(configFile)) {
```

Comment:
why is this required? you can pass a FileInputStream directly to the fromInputStream() method.

Reply:
you're right, I thought of this as a way to not having to handle the stream but it ends up just adding overhead. I'll remove it.